examples/example_hmccreds.yaml

```diff
@@ -42,6 +42,12 @@ examples:
       lparname: PART8
       #loglevel: debug
       #timestats: yes
+   example7:
+      hmc: "9.152.133.209"
+      cpcname: DPMSE2
+      cpcstatus: Active
+      partitionname: sree_test
```

Comment:
In example 5 we use partname instead of partitionname. To be consistent use partname or change both to partitionname.